.github/workflows/sync-from-snapshot-mainnet-paritydb.yml

```diff
@@ -0,0 +1,69 @@
+---
+# This workflow performs sync to Mainnet from a ParityDB non-pruned snapshot using the latest
+# main version.
+#
+# For now, this test not quite correctly tests sync Mainnet from latest aleph-node binary,
```

Comment:
Since these tests now have the additional purpose of marking latest snapshots perhaps they could just be run with the current testnet code rather than tip of `main` (which _should_ actually work)? Is this somehow harder to organize than I think?

Reply:
Yes, let's do this, but not yet in this PR, as it is already large. It can be easily done using the `get-fqdn-image` action that other workflows use. We'll save 15 minutes by not building `aleph-node`, as this action checks if the `aleph-node` docker image already exists on ECR.